### ISSUE TITLE
'FLEX/FLEXImagePreviewViewController.h' file not found

### ISSUE THREAD

**Issue by NicolasKim** [meitu/MTHawkeye#67]
In Swift project   

   ```
    pod 'MTHawkeye', :configurations => 'Debug'

    pod 'FLEX', :configurations => ['Debug']
    pod 'FBRetainCycleDetector', :configurations => ['Debug']
    pod 'fishhook', :configurations => ['Debug']
    pod 'CocoaLumberjack', '3.6.0', :configurations => ['Debug'] # CocoaLumberjack is optional, change to `MTHawkeye/DefaultPluginsWithoutLog` if don't need.
    # pod 'MTGLDebug', :configurations => ['Debug'] # MTGLDebug is exclude by default, change `MTHawkeye` to `MTHawkeye/DefaultPlugins` to include.

    pod 'MTAppenderFile', :configurations => ['Debug']
```

Build fail: 'FLEX/FLEXImagePreviewViewController.h' file not found

**Comment by haowenwu** [comment]
check FLEX is 4.1.1

**Comment by Phoenix7351** [comment]
Having the same problem. FLEX is 4.1.1

**Comment by haowenwu** [comment]
FIX at #69
